themes/bootstrap3/templates/search/reservessearch.phtml

```diff
@@ -18,7 +18,7 @@
     <input id="reservesSearchForm_lookfor" type="text" name="lookfor" size="40" value="<?=$this->escapeHtmlAttr($reservesLookfor)?>" <?=$this->searchOptions('SolrReserves')->autocompleteEnabled() ? ' class="autocomplete searcher:SolrReserves type:Reserves"' : ''?> />
     <input class="btn btn-primary" type="submit" name="submit" value="<?=$this->transEsc("Find")?>"/>
   </form>
-  <script type="text/javascript">$("#reservesSearchForm_lookfor").focus()</script>
+  <?php $this->headScript()->appendScript("$('#reservesSearchForm_lookfor').focus()");?>
```

Comment:
Looks like this is another inline script (unintentionally?) changed to a head script in the refactoring. I think I may have just missed this on my first review.

Reply:
Fixed in follow-up

Reply:
Thanks!